src/Controls/tests/DeviceTests/ControlsHandlerTestBase.cs

```diff
@@ -150,6 +140,9 @@ protected Task CreateHandlerAndAddToWindow<THandler>(IElement view, Func<THandle
 					{
 						IView content = window.Content;
 
+						if (window is Window w && w.Navigation.ModalStack.Count > 0)
+							content = w.Navigation.ModalStack.LastOrDefault();
```

Comment:
content is set to not nullable (no idea if we have nullability in this file) gout LastOrDefault can return null, should we worry?

Reply:
I'll change this to `Last`